templates/web/pages/session/sign_in_form.tt.html

```diff
@@ -2,6 +2,38 @@
 
 [% INCLUDE 'web/common/includes/error_list.tt.html' %]
 
+<!-- Add Font Awesome CSS -->
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
```

Comment:
We want to avoid using CDN as they track people, better add it to the assets (in html/fonts). I think you can add the link to load the font in the base template.

Also we already have material font (`material-icons-v139`) [which provides "Visibility" and "Visibility off" symbols](https://fonts.google.com/icons?icon.query=visibility), it would be better to stick to those.

Eg:
```html
<span class="material-icons material-symbols-visibility">
```